src/install/installer.ts

```diff
@@ -16,18 +16,12 @@
 
 import fs from 'fs';
 import path from 'path';
-import util from 'util';
 import lockfile from 'proper-lockfile';
 import {Registry, allBrowserNames, isBrowserDirectory, BrowserName, registryDirectory} from '../utils/registry';
 import * as browserFetcher from './browserFetcher';
 import { getAsBooleanFromENV, calculateSha1, removeFolders } from '../utils/utils';
 
-const fsMkdirAsync = util.promisify(fs.mkdir.bind(fs));
-const fsReaddirAsync = util.promisify(fs.readdir.bind(fs));
-const fsReadFileAsync = util.promisify(fs.readFile.bind(fs));
-const fsExistsAsync = (filePath: string) => fsReadFileAsync(filePath).then(() => true).catch(e => false);
-const fsUnlinkAsync = util.promisify(fs.unlink.bind(fs));
-const fsWriteFileAsync = util.promisify(fs.writeFile.bind(fs));
+const fsExistsAsync = (filePath: string) => fs.promises.readFile(filePath).then(() => true).catch(e => false);
```

Comment:
Not your code, but this is a really funny way to do fs.exists.

Reply:
cc @aslushnikov maybe you are interested in a nit pick (rename or change it to fs.stat) since it was introduced by one of your patches: https://github.com/microsoft/playwright/pull/3008/files#diff-0b52ef30133aabfbf9ddde0aaa90370d351c36157422fbb08cd38c12be7b8f1dR29